packages/assetpack/src/image/mipmap.ts

```diff
@@ -22,7 +22,7 @@ const defaultMipmapOptions: Required<MipmapOptions> = {
     fixedResolution: 'default',
 };
 
-export function mipmap(_options: MipmapOptions = {}): AssetPipe<MipmapOptions, 'fix'>
+export function mipmap(_options: MipmapOptions = {}): AssetPipe<MipmapOptions, 'fix' | 'nomip'>
```

Comment:
does `fix` not do the same as  'nomip'?

Reply:
`fix` resizes the image the to `fixedResolution` so if you provide `resolutions: { high: 2, default: 1, low: 0.5 }` it would resize to `default`